features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumRawEvent.kt

```diff
@@ -22,14 +22,14 @@ internal sealed class RumRawEvent {
     abstract val eventTime: Time
 
     internal data class StartView(
-        val key: Any,
+        val key: RumScopeKey,
```

Comment:
Should we rename `RumScopeKey` to `RumViewScopeKey`? Because `key` notation is used for some other classes of `RumRawEvent`, but we are changing it to `RumScopeKey` only for `StartView` and `StopView`.